LICENSES/LicenseRef-Slint-commercial.md

```diff
@@ -1,256 +1,106 @@
+Slint is a toolkit that can be used to build user interfaces for applications.
+Slint (hereafter referred to as "Software") is made available under
+different licenses by SixtyFPS GmbH incorporated at Oranienburger Str. 44, 16540
+Hohen Neuendorf, Germany ("SixtyFPS", "us", "we" or "our").
 
-SixtyFPS GmbH with offices at Oranienburger Str. 44, 16540 Hohen Neuendorf,
-Germany (**SixtyFPS**, **us**, **we** or **our**) grants a license to the
-software (**Licensed Software**), as specified herein, subject to the Terms and
-Conditions of this Slint Software License Agreement (**Agreement**).
+The "Slint Software License" is suitable for those who develop applications and
+do not want to use Slint open source components under copyleft licenses.
 
-By using the Licensed Software, **you** agree to be bound by this Agreement.
-**If you do not agree, do not use the Licensed Software.** If you are accessing
-and using the Licensed Software on behalf of a company (such as your employer)
-or other legal entity, you represent and warrant that you have the authority to
-bind that company or other legal entity to this Agreement. In that case, **you**
-and **your** will refer to that company or other legal entity.
+By using the Software, You agree to be bound by this license.
+**If you do not agree, do not use the Software.**
 
-Each Party to this Agreement may be referred to herein individually as a
-**Party** or collectively as the **Parties**.
+If you are accessing and using the Software on behalf of a company (such as your
+employer) or other legal entity, you represent and warrant that you have the
+authority to bind that company or other legal entity to this Agreement. In that
+case, "You" and "Your" will refer to that company or other legal entity.
 
-## Licensed Software
+Each Party to this license may be referred to herein individually as a
+"Party" or collectively as the "Parties".
 
-The Licensed Software includes
+### 1. Grant of Rights
 
-1. Slint compiler tool to translate Slint Design Markup Language files to Rust
-    (last stable Rust version at the time of licensed Slint version) and C++
-    (second last C++ version at the time of licensed Slint version),
+SixtyFPS hereby grants You a world-wide, non-exclusive, non-transferable license
+to use, reproduce, make available, modify, display, perform, distribute the
+Software as part of a Desktop, Web, Mobile, or Embedded System Application.
 
-2. Slint run-time library that can render interactive user-interfaces and
-    provides Rust and C++ APIs (as per above versioning) to control the
-    interfaces,
+A Desktop Application is a computer program that is designed to run on a
+general-purpose computer (PC or notebook), typically installed and executed
+locally on the computer's operating system.
 
-3. API Documentation for Slint and reference documentation for Slint Design
-    Markup Language, and
+A Web Application is a computer program that is running on a server and accessed
```

Comment:
Are you sure about the "running on a server" part? For Slint compiled to web-assembly there is nothing implied about anything running on a server.

Maybe it would make more sense to say that it's designed to run in the sand box environment provided by web browsers. (what about web runtimes?)